tests/unit/amo/components/TestLandingAddonsCard.js

```diff
@@ -106,6 +107,77 @@ describe(__filename, () => {
     expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
   });
 
+  it('hides the footer link when there are less add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [
+      createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        slug: 'custom-addon',
+      }),
+    ];
+    const root = render({ addons, isTheme: true });
+    expect(root.find(AddonsCard)).toHaveProp('footerLink', null);
+  });
+
+  it('shows the footer link when there are more or as many add-ons than LANDING_PAGE_THEME_COUNT', () => {
+    const addons = [
```

Comment:
As above, we shouldn't assume the test knows how many add-ons to create, but should base that number on the actual value of `LANDING_PAGE_THEME_COUNT`.